### PR TITLE
edit lesson added

### PR DESCRIPTION
**Pull request by Winchestery**
Mock-up
image1

HTML/CSS Layout
image2

### CODE REVIEW

**Review by MarinaPasternak — APPROVED** [review]

File: src/features/add-lesson/add-lesson.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const AddLesson = () => {
     return (
         <div className={ classNames( styles.page, 'container')}>
             <div className="row">
-                <form id='form'>
+                <form id='form' className={`${styles.size}`}>
```
Comment (MozdolevskyiMaksym) [inline src/features/add-lesson/add-lesson.js:27]: className={styles.size}

File: src/features/edit-lesson/edit-lesson.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from './edit-lesson.module.scss';
```
Comment (MozdolevskyiMaksym) [inline src/features/edit-lesson/edit-lesson.js:3]: `import styles from './edit-lesson.scss';`

**Review by MozdolevskyiMaksym — APPROVED** [review]

File: src/features/add-lesson/add-lesson.js (hunk 1 above)
Comment (Winchestery) [reply src/features/add-lesson/add-lesson.js]: done

File: src/features/edit-lesson/edit-lesson.js (hunk 2 above)
Comment (Winchestery) [reply src/features/edit-lesson/edit-lesson.js]: done

**Review by sarhan-azizov — APPROVED** [review]

**Pull request merged**